src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1964,6 +1964,23 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundConvertedStackAllocExpression
+    {
+        protected override OperationKind ExpressionKind => OperationKind.None;
+
+        protected override ImmutableArray<IOperation> Children => ImmutableArray.Create<IOperation>(this.Count);
+
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitNoneOperation(this);
+        }
+
+        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
+        {
+            return visitor.VisitNoneOperation(this, argument);
```

Comment:
Let's take a note to notify IOperation folks. They should have a backlog to add this to.

Reply:
Filed #21921 for that.

---
In reply to: [136684070](https://github.com/dotnet/roslyn/pull/21746#discussion_r136684070) [](ancestors = 136684070)